pkg/report/output/policies/policies.go

```diff
@@ -124,7 +124,7 @@ func BuildReportString(policyResults map[string][]PolicyResult, policies map[str
 
 	writePolicyListToString(reportStr, policies)
 
-	breachedPolicies := map[string]map[string]bool{
+	failedPolicies := map[string]map[string]bool{
```

Comment:
Wondering if `policyFailures` makes more sense and aligns better with `policy_failure` that we use elsewhere. What do you think? 

I probably should have used `policyBreaches` here for the same reason... 😅 

```suggestion
	policyFailures := map[string]map[string]bool{
```

Reply:
:+1: Good idea